avm/res/web/static-site/main.bicep

```diff
@@ -301,7 +327,15 @@ module staticSite_privateEndpoints 'br/public:avm/res/network/private-endpoint:0
         'Full'
       ).location
       lock: privateEndpoint.?lock ?? lock
-      privateDnsZoneGroup: privateEndpoint.?privateDnsZoneGroup
+      privateDnsZoneGroup: {
```

Comment:
Please note. the Interface still allows the user to define a custom `privateDnsZoneGroup` even though the module would ignore it.
Either this module should implement its on PE interface and essentially not allow the user to set the `privateDnsZoneGroup` - or - this could be changed to something like
```bicep
privateDnsZoneGroup: privateEndpoint.?privateDnsZoneGroup ?? {
```
To, in theory still allow the user to bring a private DNS Zone. Otherwise it would appear quite confusing that the parameter is there, but its value would be thrown away if you catch my drift 😄

Reply:
Come to think of it, it would probably be a good call to call this also out in the description of the `privateEndpoints` parameter

Reply:
This is the intended behaviour, in most cases DNS Zones are stored centrally so the user will want to BYODnsZone ...

However I have updated the PE Description now

Reply:
Fixed as discussed